community/kernel/src/main/java/org/neo4j/kernel/impl/api/ConstraintEnforcingEntityOperations.java

```diff
@@ -270,6 +273,20 @@ public PrimitiveLongIterator nodesGetFromIndexSeek( KernelStatement state, NewIn
     }
 
     @Override
+    public PrimitiveLongIterator indexQuery( KernelStatement statement, NewIndexDescriptor index,
+                                             IndexQuery[] predicates ) throws IndexNotFoundKernelException
+    {
+        return entityReadOperations.indexQuery( statement, index, predicates );
+    }
+
+    @Override
+    public PrimitiveLongIterator nodesGetFromCompositeIndexSeek( KernelStatement state, NewIndexDescriptor index,
+            CompositeIndexQuery query ) throws IndexNotApplicableKernelException, IndexNotFoundKernelException
+    {
+        return entityReadOperations.nodesGetFromCompositeIndexSeek( state, index, query );
```

Comment:
Looks like this could be removed, once indexQuery() supports multiple predicates